blocks/callout/callout.js

```diff
@@ -55,16 +56,15 @@ export default async function decorate(block) {
   } else if (calloutData.url && !calloutData.buttonLabel) {
     // The callout as a whole is interactive
     calloutBlock = document.createElement("a");
-    calloutBlock.classList.add("callout--with-link");
+    calloutBlock.classList.add("callout", "callout--with-link");
     calloutBlock.href = calloutData.url;
     if (calloutData.altText) calloutButton.setAttribute("aria-label", calloutData.altText);
   } else {
     // The callout is non-interactive
     calloutBlock = document.createElement("div");
+    calloutBlock.classList.add("callout");
```

Comment:
this change is so that the modifier class is after the main class in hierarchy and thus takes precedent in the cascade